xml/docu_styleguide.structure.xml

```diff
@@ -2535,6 +2535,74 @@ xml:id="tab-install-source"</screen>
 <screen>&lt;phrase role="productname"&gt;LibreOffice&lt;/phrase&gt; is an office suite.</screen>
  </sect2>
 
+ <sect2 xml:id="sec-profiling">
+  <title>Profiling</title>
+  <para>
+   Profiling is convenient for the creation of consistent documentation 
+   across different products or product lines. This is especially beneficial 
+   when similar products share a considerable amount of features, with only 
+   minor differences. Instead of maintaining separate documentation for
+   each product, you can share most of the XML source code and only vary text 
+   snippets where necessary. In DocBook XML files, you can mark some elements
+   as conditional by using profiling attributes. When processing the files to 
+   generate output, specify which conditions apply to the output. The style sheets
+   will then include or exclude the marked text, according to the conditions.</para>
+   <para>Profiling allows you to keep both common and product-specific content in 
+   one XML file and select at production time which information to include in 
+   the output.</para>
+  <para>If you need to use profiling in your writing, adhere to the following guidelines:</para>
+  <itemizedlist>
+   <listitem>
+    <para>
+     Identify different variants that you want to apply to the general piece of text
+     and assign clear and short identifiers to them, sticking to lowercase.
+     These identifiers act as <quote>aliases</quote> for longer products or 
+     deliverables.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+      Select one or more profiling attributes and add them to the text snippets that 
+      are conditional. The tagged snippets will only be included in the output if 
+      the required condition is fulfilled. In most cases, the attribute to use is 
+      <literal>os</literal>. For different processor architectures, use 
+      <literal>arch</literal>. The general-purpose attribute is <literal>condition</literal>.</para>
+   </listitem>
+   <listitem>
+    <para>
+      Mark the variants in your text with the relevant identifiers. Any content that 
+      is valid for all conditions does not need any profiling attributes. The 
+      respective content will always be included in the output formats generated 
+      from the XML sources. </para>
+   </listitem>
+   <listitem>
+    <para>
+      Create a different DC file for each variant. Add the respective profiling 
+      variable and its value to the DC file.</para>
+   </listitem>
+  </itemizedlist>
+  <example xml:id="ex-profiling-xml">
+    <title>A phrase in the XML file with profiling attributes</title>
```

Comment:
This is an example for _single profiling_ with _one_ attribute. Maybe reflect that in the title?

It's _one_ profiling attribute (singular!). I would also remove the "in the XML file" because where else should it be? :wink: 


```suggestion
    <title>A phrase with one profiling attribute</title>
```